cloud_governance/policy/aws/cost_explorer_payer_billings.py

```diff
@@ -22,6 +26,10 @@ def __init__(self):
         self.__ce_client = boto3.client('ce', aws_access_key_id=self.__access_key, aws_secret_access_key=self.__secret_key, aws_session_token=self.__session)
         self.__cost_explorer_operations = CostExplorerOperations(ce_client=self.__ce_client)
         self.__replacement_account = literal_eval(self._environment_variables_dict.get('REPLACE_ACCOUNT_NAME'))
+        self.__savings_discounts = float(self._environment_variables_dict.get('PAYER_SUPPORT_FEE_CREDIT', 0))
+        self.__monthly_cost_for_spa_calc = {}
+        self.__monthly_cost_for_support_fee = {}
+        self.__temporary_dir = self._environment_variables_dict.get('TEMPORARY_DIR', '/tmp')
```

Comment:
I think you should add it to [environment_variables]( https://github.com/redhat-performance/cloud-governance/blob/main/cloud_governance/main/environment_variables.py)

Reply:
Already added it to environment_variables, but added it here too.
Now removed.